src/constants.ts

```diff
@@ -12,22 +12,48 @@ import pythonQuiz from "./data/python-quiz";
 import qualityAssuranceQuiz from "./data/quality-assurance-quiz";
 import securityQuiz from "./data/security-quiz";
 import sqlQuiz from "./data/sql-quiz";
+import { QuizQuestion } from "./types";
+
+export const CATEGORY_NAMES = {
+  ACCESSIBILITY: "Accessibility",
+  AGILE: "Agile",
+  CSS: "CSS",
+  FREECODECAMP: "freeCodeCamp",
+  GENERAL: "General Computer Science",
+  GIT: "Git",
+  HTML: "HTML",
+  JAVASCRIPT: "JavaScript",
+  INFOTECH: "Information Technology",
```

Comment:
Let's switch these so it is in alphabetical order since one of my refactored suggestions in the Results file consists of creating an array from this object. 

```suggestion
 INFOTECH: "Information Technology",
  JAVASCRIPT: "JavaScript",
```